lib/simple-profiler.js

```diff
@@ -6,12 +6,15 @@ const { EventEmitter } = require('events');
 const XError = require('xerror');
 const Table = require('easy-table');
 const _ = require('lodash');
+const LRU = require('lru-cache');
 
 const ProfilerBlock = require('./profiler-block');
 
 global.simpleProfiler = global.simpleProfiler || {};
 const isEnabledSymbol = Symbol.for('zs-simple-profiler:isEnabled');
 
+const MAX_BLOCKS = 200;
```

Comment:
I can imagine valid cases with more than 200 blocks at once; maybe make the default for this 1000?